ci-operator/config/openshift/lightspeed-operator/openshift-lightspeed-operator-main.yaml

```diff
@@ -110,6 +110,9 @@ tests:
 - as: security
   optional: true
   steps:
+    env:
+      PROJECT_NAME: lightspeed-operator
```

Comment:
any reason you needed to add this?

see the discussion i had w/ TP about this here:
https://redhat-internal.slack.com/archives/CBN38N3MW/p1716315436075249?thread_ts=1716314770.838859&cid=CBN38N3MW

i'd rather let them default it than us set it(so that if at some point in the future the expected value changes, we get it for free), unless there is a reason we think the default is wrong.

Reply:
Yes, let's use default project name. New commit is pushed.